LR_Tourist/TouristWebAPI/Controllers/UserController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BLL.Model;
+using BLL.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace TouristWebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    public class UserController : Controller
```

Comment:
This controller not operating Auth data?
If yes - remove this.
No authorization in API.

Reply:
fixed Autorization